pkg/get/get.go

```diff
@@ -284,3 +289,73 @@ func GetBinaryName(tool *Tool, os, arch, version string) (string, error) {
 
 	return "", errors.New("BinaryTemplate is not set")
 }
+
+// DownloadList verifies tool exists and returns a download list
+func DownloadList(tools Tools, toolArgs []string, version string) (Tools, error) {
```

Comment:
It seems like this method isn't named correctly, it's called Download List which implies it downloads a series of tools from a list.

Instead it builds a list of URLs for downloading later on.

A slight tweak would make this more understandable in the long run:

```go
BuildDownloadURLs
GetDownloadURLs

```